src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -170,6 +137,71 @@ const ProjectExternalSubmissionsTable = ({
     beforeFormComponent: valuesViewComponent,
   });
 
+  const getColumnDefs = useCallback(() => {
+    return [
+      {
+        header: 'ID',
+        render: (s: ExternalFormSubmissionSummaryFragment) => s.id,
+      },
+      {
+        header: 'Status',
+        linkTreatment: false,
+        render: (s: ExternalFormSubmissionSummaryFragment) => {
+          const isNew = s.status === ExternalFormSubmissionStatus.New;
+          return (
+            <>
+              <Chip
+                label={capitalize(s.status)}
+                size='small'
+                color={isNew ? 'primary' : 'default'}
+                variant={isNew ? 'filled' : 'outlined'}
+                sx={isNew ? {} : { color: theme.palette.text.secondary }}
+              />
+              {s.spam && (
+                <Chip
+                  label='Spam'
+                  size='small'
+                  color='error'
+                  variant='outlined'
+                  sx={{ ml: 1, color: theme.palette.error.dark }}
+                />
+              )}
+            </>
+          );
+        },
+      },
+      {
+        header: 'Date Submitted',
+        linkTreatment: false,
+        render: (s: ExternalFormSubmissionSummaryFragment) => {
+          const parsedDate = parseHmisDateString(s.submittedAt);
+          if (parsedDate) return formatRelativeDateTime(parsedDate);
+          return '';
+        },
+      },
+      {
+        header: 'Action',
+        render: (s: ExternalFormSubmissionSummaryFragment) => (
+          <Button
```

Comment:
I think it's probably best to disable this button while the bulk mutation is loading. Then we don't need to worry about the situation where a user initiates a bulk action, and while that's loading, they also perform an individual action.